react/components/Billing/components/BillingForm/index.js

```diff
@@ -322,7 +340,7 @@ class BillingForm extends PureComponent {
                 onClick={this.handleSubmit}
                 disabled={(
                   (!customer.default_credit_card) ||
-                  (operations.length === 0)
+                  (operations.length === 0 && !this.props.plan_id)
```

Comment:
I'm not sure I follow the reasoning here? Can you explain?

Reply:
Taking this out